core/modules/translation/tests/translation.test

```diff
@@ -417,6 +417,7 @@ class TranslationTestCase extends BackdropWebTestCase {
    *   TRUE if the language switch links are found, FALSE if not.
    */
   function assertLanguageSwitchLinks($node, $translation, $find = TRUE, $types = NULL) {
+    $site_config = config('system.core');
```

Comment:
Again... :wink:  ... or your set a variable like:
```
$language_default = config_get('system.core', 'language_default');
```
As this language_default is the only config item needed here.

Reply:
Thanks for noting this one too. Changed.